.luarc.json

```diff
@@ -8,6 +8,7 @@
       "${3rd}/busted/library"
     ],
     "ignoreDir": [
+      "/lua/vim/async.lua",
```

Comment:
async.lua is typechecked with emmylua due to is much stronger support for generics